rev_news/drafts/edition-36.md

```diff
@@ -101,6 +101,8 @@ __Light reading__
 
 __Git tools and sites__
 
+* [GitFinder](https://gitfinder.com) is a new Git client for Mac that provides status badges and context menus directly into the Finder.
```

Comment:
Actually GitFinder was already mentioned in edition 26, so I removed "new" before "Git client" in https://github.com/git/git.github.io/commit/a4b5f7968e6d9e95862ebbbdc1955ada81dcd7ba